.circleci/handle_request.sh

```diff
@@ -13,7 +13,10 @@ if [ -z "$PROCESSED_UNCHANGED" ]; then
 fi
 
 # Exit if no yaml files in "submitted/" directory
-if [ ! -f submitted/*.yml ] && [ ! -f submitted/*.yaml ]; then
+shopt -s nullglob dotglob
+
+files=(submitted/*.y*ml)
```

Comment:
Using `dotglob` is a great idea! But as I mentioned, this will allow a user to submit a file like `foo.yaaaaml`. Here is a solution that I just tested:
```shell
...
shopt -s nullglob dotglob

yml_files=(submitted/*.yml submitted/*.yaml)
if [ ${#yml_files[@]} -eq 0 ]; then
    exit 0
fi

# Name of flag file, whose existence indicates that request has been handled correctly.
INDICATOR_FILENAME="submitted_data/request.handled"

# Remove flag file before data processing
rm -rf $INDICATOR_FILENAME

# For each yaml file, download and process data
for f in ${yml_files[*]}; do
    cimr processor -process -yaml-file $f
done

...
```

Reply:
A good point. thank you!